examples/dialog-modal/js/dialog.js

```diff
@@ -126,12 +126,20 @@ aria.Utils = aria.Utils || {};
    */
   aria.Dialog = function (dialogId, focusAfterClosed, focusFirst) {
     this.dialogNode = document.getElementById(dialogId);
-    var role = this.dialogNode.getAttribute('role');
-    var isDialog = role.split(' ').some(function (r) {
-      return r === 'dialog' || r === 'alertdialog';
-    });
+    if (this.idalogNode === null) {
```

Comment:
`idalogNode` looks like a typo.

Reply:
Potentially you could shorthand this into a one-liner
`(this.dialogNode = document.getElementById(dialogId)) || throw new Error('No element found with id="' + dialogId + '".');`

Reply:
🤦‍♂️ fixed that typo.

I like your error handler suggestion, but am going to leave it as-is for consistency's sake&mdash;most other errors are thrown with explicit if/else patterns.